icicle-compiler/test/cli/icicle/t1-dictionary.toml

```diff
@@ -11,5 +12,14 @@ import = []
     action="string*"
     admitted="time*"
 
+[feature.mean_severity]
+ expression = """
+   feature injury ~> mean severity
+ """
+
 [feature.mean_salary]
- expression = "feature salary ~> mean value"
+ namespace = "employee-output"
+ expression = """
+   feature "employee-input:salary"
+     ~> mean value
+ """
```

Comment:
yep, it is optional in the query language for now, but the plan is for napoleon to always output qualified names eventually